app/src/views/home/home_screen.tsx

```diff
@@ -273,6 +274,16 @@ export default function HomeScreen({
             navigateToCamera();
           }
         }}
+        rightAction={() => {
+          if (userIsGuest) {
+            setShowGuestPopup("Camera");
+          } else {
```

Comment:
You could adapt the popUp to not match Camera but instead something like "create your own memories by signing up"